tests/platform_tests/args/counterpoll_cpu_usage_args.py

```diff
@@ -0,0 +1,11 @@
+def add_counterpoll_cup_usage_args(parser):
```

Comment:
add_counterpoll_**cup**_usage_args to add_counterpoll_cpu_usage_args

Reply:
fix